### ISSUE TITLE
Auto-Sorting Feature stopped working on Ultimate Tier Factories

### ISSUE THREAD

**Issue by AndreCelso** [sddsd2332/MEK-1.12.2-v10#36]
When a lot of items get into the Ultimate Factories they don't get sorted and equally spread across all the available slots, remaining on a single slots, which prevents the user from using the machine to its fully potential

i'm not sure what did i do for this to happen, but my auto-sorting is not working anymore. 
Turning it ON and OFF multiple times to make it come back to work also doesn't work 

Forge: 14.23.5.2860
Mekanism Version: MekanismAll-1.12.2-9.8.11.150-sddsd2332.jar

**Comment by sddsd2332** [comment]
Did you use shift + mouse to pick up items?
If you do this
You will need to manually rearrange the items into the input slots

**Comment by sddsd2332** [comment]
At the moment I only know
If you use the mouse to organize the input slots while SHIFT is pressed, it may occur that sorting does not automatically update the input slots

**Comment by sddsd2332** [comment]
When there are multiple different items, one of the items will be sorted first, instead of counting the type of item before sorting?

**Comment by AndreCelso** [comment]
> Are there any images or videos?

yeah, i just clipped a moment in my stream on twitch where that happens. i hope it can show properly the issue i'm having with it
https://clips.twitch.tv/EnticingBlitheDragonflySquadGoals-ds3ellKnkrzD4RNk

**Comment by AndreCelso** [comment]
> At the moment I only know If you use the mouse to organize the input slots while SHIFT is pressed, it may occur that sorting does not automatically update the input slots

what you mean? do you mean that if i use Shift + Mouse Click to manually sort the items, it will make it work again?
or did you mean that doing that will actually cause the issue?

**Comment by AndreCelso** [comment]
> When there are multiple different items, one of the items will be sorted first, instead of counting the type of item before sorting?

what you mean? when the items get into the machines, even if there are multiple tipes of items, they just stack in a single slot instead of spreading to the other available slots

**Comment by sddsd2332** [comment]
Can you upload modpacks and world to git?
